src/include/txn.h

```diff
@@ -140,6 +140,31 @@ struct __wt_txn_shared {
     WT_CACHE_LINE_PAD_END
 };
 
+/*
+ * WT_TXN_PREPARED --
+ *	A structure to store the transactions prepared operations.
+ */
+struct __wt_pending_prepared_item {
+    TAILQ_ENTRY(__wt_pending_prepared_item) hashq;
+    uint64_t prepared_id;
+    WT_TXN_OP *mod;
+    size_t mod_alloc;
+    u_int mod_count;
```

Comment:
We define those as u_int in __wt_txn as well. May be we should change them to uint32_t?

Reply:
done